test/e2e/common.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package e2e_test
+package e2e
```

Comment:
Why we need this commit and what was wrong with keeping the e2e test utils in the `e2e_test` pkg? If there is  no particular reason, I would vote to revert the pkg change.